composer.json

```diff
@@ -10,9 +10,10 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^7.3",
+        "php": "^7.4 || ^8.0",
+        "friendsofsymfony/oauth-server-bundle": ">2.0.0-alpha.0 ^2.0@dev",
         "setono/doctrine-orm-batcher-bundle": "^0.3.1",
-        "sylius/sylius": "^1.3",
+        "sylius/sylius": "^1.9",
```

Comment:
```suggestion
        "sylius/sylius": "~1.9.0",
```

Reply:
Why would you restrict it ? I believe we can keep 1.9 and more, don't you think ?

Reply:
I like the constraint for Sylius because then when we return to this plugin one year from now we know which version worked. Sylius often times breaks between minor releases because of all sorts of config changes. This way the test application will always work